<launch>

    <arg name='host' default='localhost'/>
    <arg name='port' default='2000'/>
    <arg name='role_name' default='ego_vehicle' />
    <arg name='bt_tree_filename' default='avp_demo.xml' />
<!--     <arg name='bt_tree_filename' default='avp_demo_full.xml' /> -->
<!--     <arg name='bt_tree_filename' default='avp_demo_reeds_shepp.xml' /> -->

    <node name="carla1s_decision_node" pkg="carla1s_decision" type="carla1s_decision_node" output="screen" >
        <param name="role_name" value="$(arg role_name)" />
        <param name="bt_tree_filename" value="$(arg bt_tree_filename)" />
    </node>

    <include file="$(find carla1s_route_planner)/launch/route_planner.launch" >
        <arg name='host' value='$(arg host)'/>
        <arg name='port' value='$(arg port)'/>
        <arg name='role_name' value='$(arg role_name)' />
    </include>
    <include file="$(find carla1s_path_tracking)/launch/path_tracking.launch" >
        <arg name='role_name' value='$(arg role_name)' />
    </include>

</launch>